examples/cli/src/main/java/io/unitycatalog/cli/UnityCatalogCli.java

```diff
@@ -67,6 +74,34 @@ public static void main(String[] args) {
         return;
       }
 
+      if (cmd.hasOption("v")) {
+        CliUtils.printVersion();
+        return;
+      }
+
+      // Explanation: https://github.com/unitycatalog/unitycatalog/pull/398#issuecomment-2325039123
+      // tldr: we already have "version" for model_version entity.
+      // In the case of no args are provided it is assumed that user wants to see library version;
+      // To allow that behaviour version does not require arg, but if it used for entity args should
+      // be checked!
+      for (Option option : cmd.getOptions()) {
+        if (option.getLongOpt().equals("version")) {
+          if (cmd.getArgs().length == 0) {
+            CliUtils.printVersion();
+            return;
+          } else {
+            if (option.getValue() == null) {
+              System.out.println(
```

Comment:
Should this go to `err` instead?

Reply:
It mimics exactly the behavior of apache commons internal validator (the error, when the arg is required but not provided). I checked it: it goes to stdout, not stderr (I tried to redirect 1>out 2>err and in both cases of my "validation" and validation from apache commons all the output is in out)